### DISCUSSION TITLE
Make an option for Android that does not rotate the screen

### DISCUSSION THREAD

**Discussion by Lunasciel** [KRTirtho/spotube#1703] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Pain

Sometimes it is very annoying if you rotate the screen and the whole look changes, and you have a double list of options that are available in the "horizontal" player.

### Suggested solution

Make a checkbox in the Settings to turn on or off the screen rotation

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by KRTirtho** [comment]
This is a system option that all android OEM OS has. You can just turn off the rotation from the Notification panel of your OS.

**Comment by Lunasciel** [comment]
But then is it deactivated globally or can it be set separately for Spotube?
